package.json

```diff
@@ -193,6 +193,7 @@
 		"fast-glob": "3.2.7",
 		"filenamify": "4.2.0",
 		"glob": "7.1.2",
+		"graceful-fs": "4.2.10",
```

Comment:
This is to solve an issue that one of the `react-native`-related dependencies is accessing an outdated version of `graceful-fs`.

TBH, I'm not entirely sure why this is only happening in this PR. It might have something to do with the split steps of `npm ci --ignore-scripts` and `npm rebuild`. 🤔